lib/Mojo/Promise.pm

```diff
@@ -283,6 +304,24 @@ reason.
     say "We are done!";
   });
 
+=head2 get_all
+
+  my $results = $promise->get_all;
+
+Start L</"ioloop"> and stop it again once the promise has been fulfilled or
+rejected, then return the results in an arrayref if it was fulfilled, or throw
+an exception with the reason if it was rejected. An exception will be thrown if
+the L</"ioloop"> is already running.
+
+=head2 get_one
+
+  my $result = $promise->get_one;
+
+Start L</"ioloop"> and stop it again once the promise has been fulfilled or
+rejected, then return the first result if it was fulfilled, or throw an
+exception with the reason if it was rejected. An exception will be thrown if
+the L</"ioloop"> is already running.
```

Comment:
Don't duplicate the same text, just link to `get_all` and document the differences. Like other similar methods do.